### PR TITLE
[UI] fix model registry UI bugs

### PR DESCRIPTION
**Pull request by ankit-db**
## What changes are proposed in this pull request?

This PR introduces some changes in MLflow's UI, including bug fixes and lint-related issues.

## How is this patch tested?

Manually

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Give a description of this change to be included in the release notes for MLflow users.

<Include real description here>

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for
Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [x] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/server/js/package.json
Comment (harupy) [inline mlflow/server/js/package.json:50]: Unused imports should be removed (https://github.com/mlflow/mlflow/pull/2442).

File: mlflow/server/js/src/common/utils/ActionUtils.js
Comment (harupy) [inline mlflow/server/js/src/common/utils/ActionUtils.js:150]:
```suggestion
        return responseText
          .replace(/<[^>]+>/gi, '')
          .replace(/\n\n+/gi, '\n')
          .trim();
```

Can we just chain methods here?

File: mlflow/server/js/src/model-registry/components/ModelListPage.js
Comment (sueann) [inline mlflow/server/js/src/model-registry/components/ModelListPage.js:33]:
```suggestion
  // Loads the initial set of models.
```

File: mlflow/server/js/src/common/utils/ActionUtils.js
Comment (ankitmathur-db) [reply mlflow/server/js/src/common/utils/ActionUtils.js:150]: Agreed this would be good to do, but I think it's best to handle this kind of refactor in a follow-up PR (this PR has gotten a bit complicated)

File: mlflow/server/js/src/model-registry/components/ModelListPage.js
Comment (ankitmathur-db) [reply mlflow/server/js/src/model-registry/components/ModelListPage.js]: Done!

**Review by dbczumar — APPROVED** [review]
LGTM from a NaN-handling perspective. We should wait for comments from other stakeholders before merging, though.

**Comment by ankitmathur-db** [comment]
Thanks @dbczumar! I believe @aarondav and @Zangr are the ones who still need to take a look, and I'll wait for their input

**Review by Zangr — APPROVED** [review]
LGTM

**Comment by ankitmathur-db** [comment]
Got LGTM from Aaron!

**Pull request merged**
